client/assets/js/board.js

```diff
@@ -3,49 +3,57 @@
 import createCanvasGame from './createCanvasGame.js';
 import Snake from './snake';
 import Apple from './apple';
+import $ from 'jquery';
 
-const DELAY = 500;
+const DELAY = 150;
 
 export default class Board {
 
 // A snake store his position (x and y) and his score
 	constructor() {
-		this.snakes = [];
 		this.context = createCanvasGame();
+		this.snakes = [];
 		this.apples = [];
 	}
 
-	newSnake(x, y){
+	newSnake(x, y) {
+		let snake = new Snake(this.context, x, y);
+		snake.draw();
 
+		// Temp : Simulate 4 bodyParts on the snake
+		snake.addBodyPart(x - 60, y);
+		snake.addBodyPart(x - 120, y);
+		snake.addBodyPart(x - 180, y);
+		// End Temp
 
-		let snake = new Snake(this.context,x,y);
-		snake.draw();
 		this.snakes.push(snake);
 	}
 
-	newApple(x,y){
-		let apple = new Apple(this.context,x,y);
+	newApple(x, y) {
+		let apple = new Apple(this.context, x, y);
 		apple.draw();
+
 		this.apples.push(apple);
 	}
 
 	render(board) {
 		setInterval(function(){
-			board.snakes[0].move()
+			board.snakes[0].move();
 		}, DELAY);
 
 		$('body').keydown(function(e) {
-			if(e.keyCode === 37) {
-				board.snakes[0].direction = 'left';
+			let snake = board.snakes[0];
+			if(e.keyCode === 37 && snake.direction != 'left' && snake.direction != 'right') {
+				snake.direction = 'left';
 			}
-			else if(e.keyCode === 38) {
-				board.snakes[0].direction = 'top';
+			else if(e.keyCode === 38 && snake.direction != 'up' && snake.direction != 'down') {
```

Comment:
`snake.direction != 'up'` might not be necessary. It just means you're trying to move to the same direction you're going to.